### DISCUSSION TITLE
Adding more benchmark comparisons

### DISCUSSION THREAD

**Discussion by anthonyalayo** [microsoft/garnet#75] [General]
Hey there,

Exciting project! Is it possible to release benchmark comparisons for more databases? I am curious how Garnet performs against DynamoDB, Aerospike, and Couchbase.

**Comment by vazois** [comment]
We don't have any plans for this, though good idea. We probably need a common bench tool that can stress each system. We have been using our own tool. I am not sure how easy it is to modify it for the above systems.
It could a task on its own since we want to fairly evaluate each system.

**Reply by anthonyalayo** [reply]
Sounds fair to me. It's currently difficult for me to tell whether garnet is out performing them or not, making a switch a hard sell.
